flake.nix

```
@@ -27,8 +27,84 @@
                         base
                         wai
                         text
+
+                        # Uncomment on local development for testing
+                        # hspec
                     ];
                 };
+
+                # Custom configuration that will start with `devenv up`
+                devenv.shells.default = {
+                    # Start Mailhog on local development to catch outgoing emails
+                    services.mailhog.enable = true;
```

```suggestion
                    # services.mailhog.enable = true;
```

Can we keep this commented out by default?